### PR TITLE
Install script for Halyard (simplifies debian install path)

### PR DESCRIPTION
**Pull request by lwander**
@duftler PTAL

### CODE REVIEW

File: InstallHalyard.sh [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+function add_apt_repositories() {
+  # Spinnaker
+  # DL Repo goes here
+  REPOSITORY_HOST=$(echo $REPOSITORY_URL | cut -d/ -f3)
+  if [[ "$REPOSITORY_HOST" == "dl.bintray.com" ]]; then
+    REPOSITORY_ORG=$(echo $REPOSITORY_URL | cut -d/ -f4)
+    # Personal repositories might not be signed, so conditionally check.
+    gpg=""
+    gpg=$(curl -s -f "https://bintray.com/user/downloadSubjectPublicKey?username=$REPOSITORY_ORG") || true
+    if [[ ! -z "$gpg" ]]; then
```
Comment (jtk54) [inline InstallHalyard.sh]: I think `-n` works as 'nonzero string length'.

File: InstallHalyard.sh [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    echo "you must manually install java 8 and then rerun this script; exiting"
+    exit 13
+  fi
+}
+
+function install_halyard() {
+  package="spinnaker-halyard"
+  sudo apt-get install -y --force-yes --allow-unauthenticated $package
+  local apt_status=$?
+  if [[ $apt_status -ne 0 ]]; then
```
Comment (jtk54) [inline InstallHalyard.sh]: I'd add some `echo`s to let people know you're going the `dpkg` route and that `apt` install failed.

File: InstallHalyard.sh (hunk 1 above)
Comment (lwander) [inline InstallHalyard.sh]: yup whoops, meant to fix this. thanks

File: InstallHalyard.sh (hunk 2 above)
Comment (lwander) [inline InstallHalyard.sh]: good thinking

**Pull request merged**